crates/eww/src/config/system_stats.rs

```diff
@@ -19,22 +20,25 @@ impl RefreshTime {
 }
 
 static SYSTEM: Lazy<Mutex<System>> = Lazy::new(|| Mutex::new(System::new()));
-static DISKS: Lazy<Mutex<sysinfo::Disks>> = Lazy::new(|| Mutex::new(sysinfo::Disks::new_with_refreshed_list()));
+static DISKS: Lazy<Mutex<(RefreshTime, sysinfo::Disks)>> =
+    Lazy::new(|| Mutex::new((RefreshTime::new(), sysinfo::Disks::new_with_refreshed_list())));
 static COMPONENTS: Lazy<Mutex<sysinfo::Components>> = Lazy::new(|| Mutex::new(sysinfo::Components::new_with_refreshed_list()));
 static NETWORKS: Lazy<Mutex<(RefreshTime, sysinfo::Networks)>> =
     Lazy::new(|| Mutex::new((RefreshTime::new(), sysinfo::Networks::new_with_refreshed_list())));
 
 pub fn get_disks() -> String {
-    let mut disks = DISKS.lock().unwrap();
-    disks.refresh_list();
-    disks.refresh();
+    let (ref mut last_refresh, ref mut disks) = &mut *DISKS.lock().unwrap();
+    disks.refresh_specifics(true, DiskRefreshKind::everything());
```

Comment:
cleanup, nothing big
```suggestion
    let (ref mut last_refresh, ref mut disks) = *DISKS.lock().unwrap();
    disks.refresh(true);
```